data/styles/clear/include/Icons.mapcss

```diff
@@ -2449,6 +2449,9 @@ area|z16[shop=greengrocer]
 node|z16[shop=hairdresser],
 area|z16[shop=hairdresser]
 {icon-image: hairdresser-m.svg;}
+node|z16[shop=sewing],
+area|z16[shop=sewing]
+{icon-image: needle_and_thread-m.svg;}
```

Comment:
need to also add this at line 2640:
node|z17-[shop=sewing],
area|z17-[shop=sewing]
{icon-image: needle_and_thread-m.svg;}

otherwise it will _only_ render the icon at z16 :)

Reply:
@pastk Apologies, I was aware that I wasn't using it but it wasn't clear to me how the icons were being pulled in and on what conditions I would need a s or m icon, so I thought I'd add both and see your thoughts.

Reply:
No problem! I guess its better just to remove the small version, I don't think it makes sense to have it for a smaller feature like a shop.

Reply:
Is it worth adding like all other POIs here? z16 with icon and z17- with icon and caption?

Reply:
That can probably be refactored tbh, but best to do it in another PR to keep this one simple :)